### ISSUE TITLE
If admins can view all contents, users allowed to manage spaces or users cannot view all content (they should be able)

### ISSUE THREAD

**Issue by funkycram** [humhub/humhub#6248]
### What steps will reproduce the problem?

Create a group.
Allow "Manage Users":
![image](https://user-images.githubusercontent.com/23310825/233306518-84921570-c43e-420f-9425-56aeee743ae1.png)

Edit a user groups and add this group allowed to manage users:
![image](https://user-images.githubusercontent.com/23310825/233306765-1d8a5812-20bf-4019-bb20-5f90ba612a55.png)

Test with the following code:
```php
        $userId = 2;
        $user = \humhub\modules\user\models\User::findOne($userId);
        var_dump($user->can(ManageUsers::class));
```

### What is the expected result?

`bool(true)`

### What do you get instead?

`bool(false)`

### Additional info

| Q                | A
| ---------------- | ---
| HumHub version   | 1.14.0-beta.2 and 1.15.0-dev

**Comment by luke-** [comment]
@funkycram  > 
> ```
>         $userId = 2;
>         $user = \humhub\modules\user\models\User::findOne($userId);
>         var_dump($user->can(ManageUsers::class));
> ```

Your code refer to the ContentContainerPermission Manager of the user, which checks Permission specified here: e.g. https://community.humhub.com/user/account/permissions

If you want to check the `ManageUsers` permission you need, e.g.:

 ```
        $user = \humhub\modules\user\models\User::findOne($2);
        var_dump((new PermissionManager(['subject' => ]))->can(ManageUsers::class));

        // or
        Yii::$app->user->can(...)
```

`Yii::$app->user->permissionManager` is different to `Yii::$app->user->getIdentity()->permissionManager`.

**Comment by luke-** [comment]
@funkycram  
> 1/ I think we have a bug here: https://github.com/humhub/humhub/blob/develop/protected/humhub/modules/user/models/User.php#L762-L763
> 
> Should I create a PR for this bug?

Yes would be great!

> 
> 2/ I understand this is not easy to change, but I think it's confusing `$user->can(ManageUsers::class)` doesn't mean that the user can manage user. For my self, I knew that, but then I forgot it as it's not obvious.

That's right. We'll give it some thought.

**Comment by funkycram** [comment]
@luke- https://github.com/humhub/humhub/pull/6263
I've changed the name of this issue.
I've added the fix in CHANGELOG.md (and not CHANGELOG-DEV.md) because I thought you might want to have the bug fixed for Humhub 1.14.1, I hope I got it right.

**Comment by luke-** [comment]
@funkycram Yes, that's very fine thank you! You can also target such bugfixes directly against Master Branch.
